MiniKeePass/Password Entry View/KeyFileViewController.swift

```diff
@@ -54,10 +54,10 @@ class KeyFileViewController: UITableViewController {
 
         if (indexPath != oldIndexPath) {
             let oldCell = tableView.cellForRow(at: oldIndexPath)
-            oldCell!.accessoryType = .none
+            oldCell?.accessoryType = .none
```

Comment:
If the currently selected key file is scrolled off screen when a new one is selected then oldCell will be returned as `nil`.